2.6/test/run

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+#
+# Test the MongoDB image.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+set -exo nounset
+shopt -s nullglob
+
+IMAGE_NAME=${IMAGE_NAME-rhscl/rh-mongodb26-centos7-candidate}
```

Comment:
This also relates to naming - https://github.com/openshift/mongodb/pull/60#discussion_r36095751